### PR TITLE
Update mapstore-migration-guide.md

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
Improve documentation with suggestions about how to update your project with log4j2. 

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: docs/developer-guide/mapstore-migration-guide.md [hunk 1]
```diff
@@ -97,6 +108,75 @@ logger.restsrv.level=  INFO
...
+Moreover, if you have some custom code, you will hae to replace  the use of `getLogger`. Example: 
+
+```java
+private static final Logger LOGGER = Logger.getLogger(MyClass.class);
+```
+
+with 
+
+```
+private static final Logger LOGGER = LogManager.getLogger(ExpiringTask.class);
```
Comment (taba90) [inline docs/developer-guide/mapstore-migration-guide.md:171]:
```suggestion
private static final Logger LOGGER = LogManager.getLogger(MyClass.class);
```

**Review by taba90 — COMMENTED** [review]
Looks good to me. Just a minor thing

File: docs/developer-guide/mapstore-migration-guide.md [hunk 2]
```diff
@@ -97,6 +108,75 @@ logger.restsrv.level=  INFO
...
+             <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j.version}</version>
+            </dependency>
+```
+
+with 
+
+```
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:140]:
````suggestion
```xml
````

**Review by taba90 — APPROVED** [review]
looks good

File: docs/developer-guide/mapstore-migration-guide.md [hunk 3]
```diff
@@ -95,7 +106,76 @@ logger.restsrv.name=it.geosolutions.geostore.services.rest
...
+
+Moreover, if you have some custom code, you will hae to replace  the use of `getLogger`. Example:
+
+```java
+private static final Logger LOGGER = Logger.getLogger(MyClass.class);
+```
+
+with
+
+```
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:170]:
````suggestion
```java
````

File: docs/developer-guide/mapstore-migration-guide.md [hunk 4]
```diff
@@ -95,7 +106,76 @@ logger.restsrv.name=it.geosolutions.geostore.services.rest
...
+private static final Logger LOGGER = LogManager.getLogger(MyClass.class);
+```
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:172]:
````suggestion
```java
````

File: docs/developer-guide/mapstore-migration-guide.md [hunk 5]
```diff
@@ -95,7 +106,76 @@ logger.restsrv.name=it.geosolutions.geostore.services.rest
...
+
+```java
+private static final Logger LOGGER = Logger.getLogger(MyClass.class);
+```
+
+with
+
+```java
+private static final Logger LOGGER = LogManager.getLogger(MyClass.class);
+```java
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:172]:
````suggestion
```
````

File: docs/developer-guide/mapstore-migration-guide.md [hunk 6]
```diff
@@ -95,7 +106,76 @@ logger.restsrv.name=it.geosolutions.geostore.services.rest
...
+
+where `LogManager` can be imported as:
+
+```
```
Comment (offtherailz) [inline docs/developer-guide/mapstore-migration-guide.md:176]:
````suggestion
```java
````

**Pull request merged**